build/gulp/tasks/stats.ts

```diff
@@ -124,9 +134,26 @@ task(
   ),
 )
 
+function readSummaryPerfStats() {
+  return _.chain(require(paths.perfDist('result.json')))
+    .mapKeys((value, key) => _.camelCase(key))
```

Comment:
MongoDB does not allow dots in keys.

Reply:
could we, please, introduce it as a comment here? just to ensure that potential refactorings won't violate this policy.